public/components/ContentPanel/ContentPanel.tsx

```diff
@@ -105,7 +107,7 @@ const ContentPanel = (props: ContentPanelProps) => (
         className={props.horizontalRuleClassName}
       />
     )}
-    <div style={{ padding: '0px 10px', ...props.bodyStyles }}>
+    <div style={{ padding: '0px 20px', ...props.bodyStyles }}>
```

Comment:
Why increase left and right padding? Shouldn't we increase the top and down padding?

Reply:
As mentioned above, since we cannot increase EuiPanel padding, we have to increase the left/right padding of `body` to 20px. 

Regarding the top/down padding, you are right, we should make them both 20px. I didn't increase them because:

1) Top padding: I set top padding in [AnomaliesLiveChart](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/45/files#diff-d0f6b17630ef8bd093c50b787e508b88R241) and [AnomaliesDistributionChart](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/45/files#diff-474a5fc2dba872f006bdc3342074dd77R149): `FlexGroup` has default padding -12px, and `EuiHorizontalRule` has default top/bottom margin 8px, changing top padding of FlexGroup to 0px can make the distance between `EuiHorizontalRule` and FlexItems in FlexGroup to 20px

2) Bottom padding: EuiPanel has default padding 16px, thus I keep it for the bottom padding.

To consolidate style changes for ContentPanel, I think I can make following changes so that all the style change can be configured in ContentPanel, and the distance between components and Panel edge or `EuiHorizontalRule` is 20px.
```
  <EuiPanel
    style={{
      paddingLeft: '0px',
      paddingRight: '0px',
      paddingTop: '20px',
      paddingBottom: '20px',
      ...props.panelStyles,
    }}
    className={props.contentPanelClassName}
  >
```

```
    <EuiFlexGroup
      style={{ padding: '0px 20px', ...props.titleContainerStyles }}
      justifyContent="spaceBetween"
      alignItems="center"
    >
```

```
    <div
      style={{
        paddingTop: '12px', // 12px here is because EuiHorizontalRule has bottom margin 8px
        paddingLeft: '20px',
        paddingRight: '20px',
        ...props.bodyStyles,
      }}
    >
      {props.children}
    </div>
```

Reply:
I'm ok about it, we can tune if any other comments come in.